examples/hr-data/src/main.rs

```diff
@@ -0,0 +1,40 @@
+// example for reading heart rate data
+
+struct Handler;
+
+#[arctic::async_trait]
+impl arctic::EventHandler for Handler {
+    async fn heart_rate_update(&self, _ctx: &arctic::PolarSensor, heartrate: arctic::HeartRate) {
+        println!("Heart rate: {:?}", heartrate);
+    }
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let mut polar = arctic::PolarSensor::new("9F8F7521".to_string())
```

Comment:
Same two comments about the ID and the "Could not connect" wording below.

Reply:
ok